sdk/communication/communication-alpha-ids/package.json

```
@@ -0,0 +1,135 @@
+    "build:browser": "tsc -p . && dev-tool run bundle",
+    "build:node": "tsc -p . && dev-tool run bundle",
+    "build:samples": "dev-tool samples publish --force",
+    "build:test": "tsc -p . && dev-tool run bundle",
+    "check-format": "prettier --list-different --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"samples-dev/**/*.ts\" \"*.{js,json}\"",
+    "clean": "rimraf dist dist-* types *.tgz *.log",
+    "execute:samples": "dev-tool samples run samples-dev",
+    "extract-api": "tsc -p . && api-extractor run --local",
+    "format": "prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"samples-dev/**/*.ts\" \"*.{js,json}\"",
+    "integration-test:browser": "dev-tool run test:browser",
+    "integration-test:node": "dev-tool run test:node-js-input -- --timeout 300000 'dist-esm/test/public/*.spec.js'",
+    "integration-test": "npm run integration-test:node && npm run integration-test:browser",
+    "lint:fix": "eslint package.json api-extractor.json README.md src test --ext .ts,.javascript,.js --fix --fix-type [problem,suggestion]",
+    "lint": "eslint package.json api-extractor.json README.md src test --ext .ts,.javascript,.js",
+    "pack": "npm pack 2>&1",
+    "test": "rimraf dist-test && npm run build:test && npm run unit-test && npm run integration-test",
+    "test:browser": "npm run build:test && npm run unit-test:browser && npm run integration-test:browser",
+    "test:node": "npm run build:test && npm run unit-test:node && npm run integration-test:node",
+    "test:watch": "npm run test -- --watch --reporter min",
+    "unit-test:browser": "dev-tool run test:browser",
+    "unit-test:node": "dev-tool run test:node-ts-input -- --exclude 'test/**/browser/*.spec.ts' 'test/**/*.spec.ts'",
+    "unit-test": "npm run unit-test:node && npm run unit-test:browser"
+  },
+  "files": [
+    "dist/",
+    "dist-esm/src/",
+    "types/communication-alpha-ids.d.ts",
+    "README.md",
+    "LICENSE"
+  ],
+  "keywords": [
+    "azure",
+    "cloud",
+    "communication"
+  ],
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "engines": {
+    "node": ">=12.0.0"
+  },
+  "homepage": "https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/communication/communication-alpha-ids/",
+  "repository": "github:Azure/azure-sdk-for-js",
+  "bugs": {
+    "url": "https://github.com/Azure/azure-sdk-for-js/issues"
+  },
+  "sideEffects": false,
+  "prettier": "@azure/eslint-plugin-azure-sdk/prettier.json",
+  "dependencies": {
+    "@azure/abort-controller": "^1.0.0",
+    "@azure/communication-common": "^2.0.0",
+    "@azure/core-auth": "^1.3.0",
+    "@azure/core-client": "^1.3.2",
+    "@azure/core-lro": "^2.2.0",
+    "@azure/core-paging": "^1.1.1",
+    "@azure/core-rest-pipeline": "^1.3.2",
+    "@azure/core-tracing": "1.0.0-preview.13",
```

Can the GA version be used?
```suggestion
    "@azure/core-tracing": "1.0.0",
```

---

Definitely. I upgraded it to 1.0.1 (along with the necessary code adjustments).